core/src/controllers/rows.js

```diff
@@ -13,6 +13,8 @@ const operators = {
   lte: '<=',
   gte: '>=',
   neq: '!=',
+  null: 'IS NULL',
+  notNull: 'IS NOT NULL',
```

Comment:
As it's query parameter if think it's best if we stay with the lowercase standard e.g. `notnull`